net-dns/coredns/coredns-1.6.9.ebuild

```diff
@@ -0,0 +1,706 @@
+	"gopkg.in/inf.v0 v0.9.1" # BSD
+	"gopkg.in/yaml.v2 v2.2.8" # Apache-2.0
+	"k8s.io/api v0.17.4" # Apache-2.0
+	"k8s.io/apimachinery v0.17.4" # Apache-2.0
+	"k8s.io/client-go v0.17.4" # Apache-2.0
+	"k8s.io/klog v1.0.0" # Apache-2.0
+	"k8s.io/utils v0.0.0-20191114184206-e782cd3c129f" # Apache-2.0
+	"sigs.k8s.io/yaml v1.1.0" # MIT && BSD
+
+	# Tests:
+	"github.com/Azure/go-autorest/autorest/mocks v0.3.0" # Apache-2.0
+	"github.com/Shopify/toxiproxy v2.1.4+incompatible" # MIT
+	"github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa" # Apache-2.0
+	"github.com/coreos/go-semver v0.2.0" # Apache-2.0
+	"github.com/dustin/go-humanize v1.0.0" # MIT
+	"github.com/evanphx/json-patch v4.2.0+incompatible" # BSD
+	"github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef" # Apache-2.0
+	"github.com/google/btree v1.0.0" # Apache-2.0
+	"github.com/gorilla/websocket v1.4.0" # BSD-2
+	"github.com/grpc-ecosystem/go-grpc-middleware v1.0.1-0.20190118093823-f849b5445de4" # Apache-2.0
+	"github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0" # Apache-2.0
+	"github.com/grpc-ecosystem/grpc-gateway v1.9.5" # BSD
+	"github.com/jonboulle/clockwork v0.1.0" # Apache-2.0
+	"github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515" # MIT
+	"github.com/kr/pretty v0.1.0" # MIT
+	"github.com/onsi/ginkgo v1.10.1" # MIT
+	"github.com/onsi/gomega v1.7.0" # MIT
+	"github.com/pkg/errors v0.9.1" # BSD-2
+	"github.com/pmezard/go-difflib v1.0.0" # BSD
+	"github.com/sirupsen/logrus v1.4.2" # MIT
+	"github.com/soheilhy/cmux v0.1.4" # Apache-2.0
+	"github.com/stretchr/objx v0.1.1" # MIT
+	"github.com/stretchr/testify v1.4.0" # MIT
+	"github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5" # MIT
+	"github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2" # MIT
+	"go.etcd.io/bbolt v1.3.3" # MIT
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15" # BSD-2
+	"k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a" # Apache-2.0
+
+	# tests of tests. not actually needed, no need to copy to LICENSE, but at this stage
+	# it is easier to just download them to avoid warnings
+	"github.com/konsorten/go-windows-terminal-sequences v1.0.2" # MIT
+	"github.com/kr/text v0.1.0" # MIT
+	"github.com/hpcloud/tail v1.0.0" # MIT
+	"github.com/fsnotify/fsnotify v1.4.7" # BSD
+	"gopkg.in/fsnotify.v1 v1.4.7" # BSD
+	"gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7" # BSD
+
+	# see also the bottom of the file
+)
+
+EGO_PN="github.com/${PN}/${PN}"
+
+inherit go-module
+
+EGIT_COMMIT="1766568398e3120c85d44f5c6237a724248b652e"
+ARCHIVE_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="A DNS server that chains middleware"
+HOMEPAGE="https://github.com/coredns/coredns"
+
+LICENSE="Apache-2.0 MIT BSD ISC MPL-2.0 BSD-2"
+SLOT="0"
+IUSE=""
+
+src_compile() {
+	go build -v -ldflags="-X github.com/coredns/coredns/coremain.GitCommit=${EGIT_COMMIT}" || die
+}
+
+src_install() {
+	dobin ${PN}
+	dodoc README.md
```

Comment:
`einstalldocs`, just so it may catch something else useful.

Reply:
done